solutions/src/main/java/ru/sberbank/school/task03/FxRequestConversionServiceImpl.java

```diff
@@ -0,0 +1,76 @@
+package ru.sberbank.school.task03;
+
+import ru.sberbank.school.task02.exception.FxConversionException;
+import ru.sberbank.school.task02.util.Beneficiary;
+import ru.sberbank.school.task02.util.ClientOperation;
+import ru.sberbank.school.task02.util.Symbol;
+
+public class FxRequestConversionServiceImpl implements FxRequestConversionService {
+
+    public Symbol getSymbol (String symbolAsString) {
```

Comment:
&#39;(&#39; is preceded with whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-628644062>*